CHANGELOG.md

```diff
@@ -1,7 +1,32 @@
 Changes by Version
 ==================
 
-1.17.0 (2019-02-26)
+1.17.1 (2020-03-17)
+-------------------
+
+Breaking changes:
+* None
+
+Other noteworthy changes:
+* No modify annotation when injecting ([#902](https://github.com/jaegertracing/jaeger-operator/pull/902), [@rubenvp8510](https://github.com/rubenvp8510))
+* Restore two minute timeout ([#962](https://github.com/jaegertracing/jaeger-operator/pull/962), [@kevinearls](https://github.com/kevinearls))
+* Added TLS configuration for collector in all-in-one ([#960](https://github.com/jaegertracing/jaeger-operator/pull/960), [@jpkrohling](https://github.com/jpkrohling))
```

Comment:
This is a bug fix for a feature introduced in the same release. We typically remove those entries as well.